src/main/java/org/folio/consortia/utils/TokenUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.folio.consortia.utils;
+
+import org.folio.consortia.domain.dto.Payload;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+
+import java.util.Base64;
+
+public class TokenUtils {
+
+  public static Payload parseToken(String token) {
+    String[] tokenParts = token.split("\\.");
+
+    if(tokenParts.length == 3) {
+      String encodedPayload = tokenParts[1];
+      byte[] decodedJsonBytes = Base64.getDecoder().decode(encodedPayload);
+      String decodedJson = new String(decodedJsonBytes);
+
+      try {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Object mapper can be declared as private static final field to not create this object each time, by the way maybe we already have some utiliti class that doing parsing the token?

Reply:
fully removed this utils, token and user id are available in execution context